src/drt/src/dr/FlexGridGraph.h

```diff
@@ -1031,7 +1031,7 @@ class FlexGridGraph
 
   frUInt4 subFromByte(frUInt4 minuend, frUInt4 subtrahend)
   {
-    return std::max(minuend - subtrahend, 0u);
+    return std::max((int) (minuend - subtrahend), 0);
```

Comment:
Doing subtraction with unsigned and then converting to int is a bit odd (the underflow still happens but the result gets reinterpreted).  It would be more natural to convert the arguments to be signed.

https://google.github.io/styleguide/cppguide.html#Integer_Types\
"In particular, do not use unsigned types to say a number will never be negative. Instead, use assertions for this."